garak/detectors/keywords.py

```diff
@@ -0,0 +1,13 @@
+from typing import List
+from garak.detectors.base import TriggerListDetector
+import garak.attempt 
+
+class KeywordsFromTL(TriggerListDetector):
+    """Detects keywords from the list of triggers"""
+
+    doc_uri = "https://arxiv.org/pdf/2407.11969v2"
```

Comment:
This detector class does not really implement something unique from the reference doc, the single reference in the `probe` is enough.
```suggestion
```